services/asset/nodes/schemas.go

```diff
@@ -124,30 +136,72 @@ func makeNodeSchema(recursiveNodes map[string]*schema.Schema) map[string]*schema
 }
 
 var dataSourceFilterSchema = map[string]*schema.Schema{
-	"parent_node_ids": {
+	"created_by": {
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Filter on the user who created the Node",
```

Comment:
can you add validation for uuids? (here and last_modified_by)

Reply:
Yes, I will add it to properties as well.